composables/regexp.ts

```diff
@@ -0,0 +1,9 @@
+export const AT_LEAST_ONE_LETTER_UPPERCASE_REGEXP = createRegExp(oneOrMore(letter.uppercase))
+export const AT_LEAST_ONE_DIGIT_REGEXP = createRegExp(oneOrMore(digit))
+
+const REQUIRED_CHARS = '!@#$%^&*...'
```

Comment:
Zamiast bezmyslnie kopiowac, to przepisz reszte tych znakow co wczesniej dales...

Reply:
Nie było to bezmyślnie skopiowane, tak skoro dałeś taki komentarz, to zrozumiałem, że takie znaki są jedynie potrzebne